addons/project/project.py

```diff
@@ -113,6 +113,18 @@ def _get_alias_models(self, cr, uid, context=None):
         """ Overriden in project_issue to offer more options """
         return [('project.task', "Tasks")]
 
+    def _count_analytic_accounts(self, cr, uid, ids, field_name, arg, context):
+        return {
+            project_id: len(self.browse(cr, uid, project_id, context=context).analytic_account_id)
+            for project_id in ids
```

Comment:
is it really required ? analytic_account_id is many2one field for project. so we always get only one analytic account for single project.